concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-356	356	Persone	people	789	PEOPLE	611	644	663	173	615	200	300
+DellaRosa-2010-417-357	357	Pescatore	fisherman	1858	FISHERMAN	661	673	509	244	626	142	215
+DellaRosa-2010-417-358	358	Pianeta	planet			563	663	482	300	547	241	444
+DellaRosa-2010-417-359	359	Pianta	plant	653	PLANT (VEGETATION)	681	687	635	181	657	125	160
+DellaRosa-2010-417-360	360	Piccione	pigeon	1853	DOVE	691	662	618	192	655	103	225
+DellaRosa-2010-417-361	361	Pistola	gun	1566	GUN	671	694	543	265	613	114	279
+DellaRosa-2010-417-362	362	Poltrona	armchair			669	687	605	213	676	106	138
+DellaRosa-2010-417-363	363	Polvere	dust	2	DUST	668	615	626	254	575	174	269
+DellaRosa-2010-417-364	364	Pomodoro	tomato	986	TOMATO	700	689	645	195	674	100	144
+DellaRosa-2010-417-365	365	Prete	priest	852	PRIEST	700	647	632	233	660	138	288
+DellaRosa-2010-417-366	366	Prigioniero	prisoner	1072	PRISONER	563	589	400	280	576	197	464
+DellaRosa-2010-417-367	367	Primavera	spring	1537	SPRINGTIME	475	526	577	230	532	315	296
+DellaRosa-2010-417-368	368	Principe	prince	2554	PRINCE (RULER)	571	633	476	219	578	175	355
+DellaRosa-2010-417-369	369	Psicologo	psychologist			591	541	687	515	679	233	552
+DellaRosa-2010-417-370	370	Quercia	oak	644	OAK	674	644	485	291	644	110	189
+DellaRosa-2010-417-371	371	Regina	queen	1306	QUEEN	603	653	509	200	600	167	350
+DellaRosa-2010-417-372	372	Romanzo	novel			584	633	591	397	600	192	420
+DellaRosa-2010-417-373	373	Sabbia	sand	671	SAND	682	647	588	167	650	118	200
+DellaRosa-2010-417-374	374	Sale	salt	1274	SALT	673	670	674	188	674	109	188
+DellaRosa-2010-417-375	375	Sarto	tailor	881	TAILOR	674	650	479	306	615	156	388
+DellaRosa-2010-417-376	376	Scarpa	shoe	1381	SHOE	700	695	659	170	650	100	128
+DellaRosa-2010-417-377	377	Scatola	box	1465	BOX	684	697	674	145	641	113	325
+DellaRosa-2010-417-378	378	Schiavo	slave	812	SLAVE	596	613	395	356	524	244	478
+DellaRosa-2010-417-379	379	Scintilla	spark	518	SPARK	600	597	453	278	515	194	381
+DellaRosa-2010-417-380	380	Scopa	broom	245	BROOM	700	676	653	192	675	100	225
+DellaRosa-2010-417-381	381	Scrivania	desk			700	697	680	280	667	110	168
+DellaRosa-2010-417-382	382	Scuola	school	673	SCHOOL	597	680	685	219	670	196	235
+DellaRosa-2010-417-383	383	Sedia	chair	1476	CHAIR	673	700	674	150	682	106	119
+DellaRosa-2010-417-384	384	Segretaria	secretary			654	648	558	400	647	141	331
+DellaRosa-2010-417-385	385	Servo	servant	255	SERVANT	550	542	350	310	479	215	500
+DellaRosa-2010-417-386	386	Soldato	soldier	1941	SOLDIER	642	673	518	306	648	150	225
+DellaRosa-2010-417-387	387	Sole	sun	1343	SUN	623	683	668	150	682	141	169
+DellaRosa-2010-417-388	388	Sorriso	smile	1451	SMILE	600	685	637	152	644	206	338
+DellaRosa-2010-417-389	389	Specchio	mirror	1171	MIRROR	650	678	616	194	662	115	156
+DellaRosa-2010-417-390	390	Stanza	room	1450	ROOM	674	667	679	194	674	146	145
+DellaRosa-2010-417-391	391	Statua	statue	1002	STATUE	691	685	540	290	579	105	241
+DellaRosa-2010-417-392	392	Stazione	station			668	680	642	268	675	135	211
```

Comment:
Perhaps we can add a STATION concept? There are already a few instances in previous lists.

Reply:
@MiraAhmedovic Could you open an issue linking this PR where we can decide whether we want to include STATION as a new concept?